frontend/src/app/components/common/comp-coordinate-input.tsx

```diff
@@ -0,0 +1,443 @@
+      }
+    } else if (coordinateType === COORDINATE_TYPES.LatLong) {
+      if ([eastingCoordinateErrorMsg, northingCoordinateErrorMsg, zoneErrorMsg].every((element) => element === "")) {
+        if (eastingCoordinate && northingCoordinate && zoneCoordinate?.value) {
+          let utm = new utmObj();
+          const latLongCoordinates = utm.convertUtmToLatLng(
+            eastingCoordinate,
+            northingCoordinate,
+            zoneCoordinate?.value,
+            "N",
+          );
+          const lat = formatLatLongCoordinate(latLongCoordinates.lat.toString());
+          const lng = formatLatLongCoordinate(latLongCoordinates.lng.toString());
+
+          setYCoordinate(lat);
+          setXCoordinate(lng);
+          syncCoordinates(lat, lng);
+        }
+        setCoordinateType(coordinateType);
+      }
+    }
+  };
+
+  const updateUtmFields = (lat: string, lng: string): { easting: string; northing: string; zone: string } => {
+    let utm = new utmObj();
+    const utmCoordinates = utm.convertLatLngToUtm(lat, lng, 3);
+    setEastingCoordinate(utmCoordinates.Easting);
+    setNorthingCoordinate(utmCoordinates.Northing);
+    setZoneCoordinate({ label: utmCoordinates.ZoneNumber, value: utmCoordinates.ZoneNumber } as Option);
+
+    return { easting: utmCoordinates.Easting, northing: utmCoordinates.Northing, zone: utmCoordinates.ZoneNumber };
+  };
+
+  useEffect(() => {
+    const lat = formatLatLongCoordinate(initYCoordinate);
+    const lng = formatLatLongCoordinate(initXCoordinate);
+
+    setYCoordinate(lat);
+    setXCoordinate(lng);
+
+    if (lat && lng && !isZeroOrEmpty(lat) && !isZeroOrEmpty(lng)) {
+      if (coordinateType === COORDINATE_TYPES.LatLong) {
+        handleGeoPointChange(lat, lng);
+      }
+
+      if (coordinateType === COORDINATE_TYPES.UTM) {
+        const { easting, northing, zone } = updateUtmFields(lat, lng);
+        if (easting && northing && zone) {
+          handleUtmGeoPointChange(easting, northing, zone);
+        }
+      }
+    }
+  }, [initXCoordinate, initYCoordinate]);
+
+  const formatUtmCoordinate = (input: string | undefined): string => {
+    const regex = /^-?\d+\.?\d*$/;
+    let result = input;
+    if (regex.exec(input ?? "")) {
+      result = input ? Number(input).toFixed(0).toString() : "";
+    }
+    return result ?? "";
+  };
+
+  const isZeroOrEmpty = (input: string | undefined): boolean => {
+    return input === "" || input === "0";
+  };
+
+  return (
+    <div
+      className="comp-details-form-row"
+      id={id}
+    >
+      <label htmlFor="coordinate-type-radio-group">Coordinates</label>
+      <div
+        id="coordinate-input-div"
+        className="comp-details-input full-width"
+      >
+        <div className="comp-details-input full-width">
+          {
+            <CompRadioGroup
+              id="coordinate-type-radio-group"
+              options={[
+                { value: COORDINATE_TYPES.LatLong, label: "Lat/Long" } as Option,
```

Comment:
Discussed this with Dragos and Vita, would like to use Latitude/Longitude instead of Lat/Long